episode-6.srt

```diff
@@ -46,7 +46,7 @@
 10
 00:00:44,100 --> 00:00:45,680
 [ru] В предыдущих сериях:
-[en] Previously on…
+[en] Previously...
```

Comment:
@scribblemaniac 
The literal meaning here: "in previous episodes:"
Also, here we can use different phrase length, because there are no lipsync in this shot. ^__^